vividus-allure-adaptor/src/test/java/org/vividus/report/allure/plugin/EmbeddedPluginTests.java

```diff
@@ -0,0 +1,70 @@
+/*
+ * Copyright 2019-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.vividus.report.allure.plugin;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import java.util.List;
+
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.Mock;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+import io.qameta.allure.Extension;
+
+@ExtendWith(MockitoExtension.class)
+class EmbeddedPluginTests
+{
+    private static final String ID = "id";
+
+    @Mock
+    private List<String> resourceList;
```

Comment:
do we really need to mock collection here?

Reply:
Why not? What is the alternative?

Reply:
I think it may cause side effects since there are too many methods (List + Collection AP) that allure may rely on and if it doesn't fail now it  may fail in the future, old post on side effect example https://stackoverflow.com/questions/18483176/mockito-mocking-an-arraylist-that-will-be-looped-in-a-for-loop
Why not to pass empty list?